monkey/infection_monkey/master/plugin_registry.py

```diff
@@ -1,3 +1,7 @@
+# isort: off
+from serpentarium import PluginLoader
```

Comment:
We need to import this in `main.py`. Once that's done, the order won't matter in this file.